x-pack/plugins/alerts/common/alert.ts

```diff
@@ -6,6 +6,11 @@
 
 import { SavedObjectAttributes } from 'kibana/server';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AlertTypeState = Record<string, any>;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AlertTypeParams = Record<string, any>;
+
```

Comment:
Can't this be a generic type with a default of `unknown` instead of `any`?

Reply:
Converting these types to `Record<string, unknown>` caused over 130 type check failures. Most of these were outside of the alerting team's code and would cause us to change a lot of external code. I'm thinking we enforce this at some other time and have created https://github.com/elastic/kibana/issues/74897 to track this.